client/cli/Cargo.toml

```diff
@@ -50,4 +50,7 @@ sp-version = { version = "5.0.0", path = "../../primitives/version" }
 tempfile = "3.1.0"
 
 [features]
+default = ["with-parity-db", "with-rocks-db"]
+with-parity-db = ["sc-client-db/with-parity-db"]
```

Comment:
Considering how the endgame is to make the ParityDB the default and eventually remove RocksDB I think it makes sense to only make the rocksdb support optional. Are there any reasonable use cases where someone would want to compile a binary with *only* RocksDB?

Reply:
These features were originally introduced for substrate-in-the-browser build, which had both backends disabled and used custom in-memory storage. As far as I know it is no longer maintained and has been deprecated in favor of smoldot, so we can remove extra features.